htdocs/GetCSS.php

```diff
@@ -93,6 +95,20 @@
     $FullPath = "$ModuleDir/css/$File";
 }
 
+$public = false;
+try {
+    $m = Module::factory($Module);
+
+    $public = $m->isPublicModule();
```

Comment:
$public = $m->isPublicModule() always return false.
if should remove this.

Reply:
different modules returned by the factory override the function, that shouldn't be the case (if it is, how is the same code in the main.php working for the login module?)

Reply:
https://github.com/aces/Loris/pull/3412